lib/relay.js

```diff
@@ -197,6 +202,7 @@ function responseHandler(filterRules, config, io) {
         logContext.userAgentHeaderSet = true;
       }
 
+      headers.authorization = {};
```

Comment:
Why the decision to initialise this with an empty object? `headers` should essentially be `{[key: string]: string}`, surely? Shouldn't this just be `delete headers.authorization`?

Reply:
I think it is indeed better than empty object.

Will fix and add tests
(The auth can be a string too, not only am object. I.e `Basic abcd==`)